src/transforms/pipelines/mod.rs

```diff
@@ -0,0 +1,198 @@
+mod expander;
+mod router;
+
+use crate::config::{
+    DataType, ExpandType, GenerateConfig, TransformConfig, TransformContext, TransformDescription,
+};
+use crate::transforms::Transform;
+use indexmap::IndexMap;
+use serde::{Deserialize, Serialize};
+
+const SAMPLE_CONFIG: &str = r#"[logs]
```

Comment:
For readability, the [indoc!](https://docs.rs/indoc/1.0.3/indoc/) macro might be useful here.